drivers/scheduler/oke/oke.go

```diff
@@ -0,0 +1,585 @@
+			CniType string `json:"cni-type"`
+		} `json:"node-pool-pod-network-option-details"`
+		NsgIds           []interface{} `json:"nsg-ids"`
+		PlacementConfigs []struct {
+			AvailabilityDomain    string      `json:"availability-domain"`
+			CapacityReservationID interface{} `json:"capacity-reservation-id"`
+			FaultDomains          interface{} `json:"fault-domains"`
+			PreemptibleNodeConfig interface{} `json:"preemptible-node-config"`
+			SubnetID              string      `json:"subnet-id"`
+		} `json:"placement-configs"`
+		Size int `json:"size"`
+	} `json:"node-config-details"`
+	NodeEvictionNodePoolSettings *struct {
+		EvictionGraceDuration           string `json:"eviction-grace-duration"`
+		IsForceDeleteAfterGraceDuration bool   `json:"is-force-delete-after-grace-duration"`
+	} `json:"node-eviction-node-pool-settings"`
+	NodeImageID            string      `json:"node-image-id"`
+	NodeImageName          string      `json:"node-image-name"`
+	NodePoolCyclingDetails interface{} `json:"node-pool-cycling-details"`
+	NodeShape              string      `json:"node-shape"`
+	NodeShapeConfig        *struct {
+		MemoryInGBs float64 `json:"memory-in-gbs"`
+		Ocpus       float64 `json:"ocpus"`
+	} `json:"node-shape-config"`
+	NodeSource struct {
+		ImageID    string `json:"image-id"`
+		SourceName string `json:"source-name"`
+		SourceType string `json:"source-type"`
+	} `json:"node-source"`
+	NodeSourceDetails struct {
+		BootVolumeSizeInGBs interface{} `json:"boot-volume-size-in-gbs"`
+		ImageID             string      `json:"image-id"`
+		SourceType          string      `json:"source-type"`
+	} `json:"node-source-details"`
+	QuantityPerSubnet int         `json:"quantity-per-subnet"`
+	SshPublicKey      string      `json:"ssh-public-key"`
+	SubnetIds         []string    `json:"subnet-ids"`
+	SystemTags        interface{} `json:"system-tags"`
+}
+
+type WorkRequest struct {
+	OpcWorkRequestId string `json:"opc-work-request-id"`
+}
+
+type WorkRequestGet struct {
+	Data struct {
+		Resources []struct {
+			Identifier string `json:"identifier"`
+		} `json:"resources"`
+		Status string `json:"status"`
+	} `json:"data"`
+}
+
+type responseWithListId struct {
+	Data []struct {
+		DisplayName  string `json:"display-name"`
+		Id           string `json:"id"`
+		FreeformTags struct {
+			K8SVersion string `json:"k8s_version"`
+		} `json:"freeform-tags,omitempty"`
+	} `json:"data"`
+}
+
+func (o *oke) String() string {
+	return SchedName
+}
+
+// Init initializes the node driver for oke under the given scheduler
+func (o *oke) Init(schedOpts scheduler.InitOptions) error {
+
+	instanceGroup := os.Getenv("INSTANCE_GROUP")
+	if len(instanceGroup) != 0 {
+		o.instanceGroupName = instanceGroup
+	} else {
+		o.instanceGroupName = "default"
+	}
+	compartmentId := os.Getenv("PX_ORACLE_compartment_id")
+	if len(compartmentId) == 0 {
+		return fmt.Errorf("compartment id not provided as env var [PX_ORACLE_compartment_id]: %s", compartmentId)
```

Comment:
There is no point to print `compartmentId` if we know its empty

Reply:
fixed